security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,5 +1,4 @@
-FROM ubuntu:16.04
+FROM istionightly/base_debug
```

Comment:
This change will go into 1.1 branch. Are we OK with using the base_debug image?
Can the scratch image satisfy our debug needs?
https://github.com/istio/istio/blob/master/security/docker/Dockerfile.node-agent#L1

Reply:
envoy is using that image - we want to use same base consistently. If there are problems with the image, the proxy will be affected as well.

Reply:
Thanks @costinm , I just checked https://github.com/istio/istio/blob/release-1.1/pilot/docker/Dockerfile.proxyv2, and it is using base_debug